pubnub-kotlin/pubnub-kotlin-api/src/iosMain/kotlin/com/pubnub/api/endpoints/objects/member/GetChannelMembers.ios.kt

```diff
@@ -1,9 +1,64 @@
 package com.pubnub.api.endpoints.objects.member
 
+import cocoapods.PubNubSwift.PubNubChannelMetadataObjC
+import cocoapods.PubNubSwift.PubNubMembershipMetadataObjC
+import cocoapods.PubNubSwift.PubNubObjC
+import cocoapods.PubNubSwift.getChannelMembersWithChannel
 import com.pubnub.api.Endpoint
+import com.pubnub.api.models.consumer.objects.PNMemberKey
+import com.pubnub.api.models.consumer.objects.PNPage
+import com.pubnub.api.models.consumer.objects.PNSortKey
+import com.pubnub.api.models.consumer.objects.member.PNMember
 import com.pubnub.api.models.consumer.objects.member.PNMemberArrayResult
+import com.pubnub.api.models.consumer.objects.member.PNUUIDDetailsLevel
+import com.pubnub.api.onFailureHandler
+import com.pubnub.api.onSuccessHandler3
+import com.pubnub.api.v2.callbacks.Consumer
+import com.pubnub.api.v2.callbacks.Result
+import com.pubnub.kmp.createPNMember
+import com.pubnub.kmp.createPubNubHashedPage
+import com.pubnub.kmp.filteredList
+import kotlinx.cinterop.ExperimentalForeignApi
+import platform.Foundation.NSNumber
 
 /**
  * @see [PubNub.getChannelMembers]
  */
-actual interface GetChannelMembers : Endpoint<PNMemberArrayResult>
\ No newline at end of file
+actual interface GetChannelMembers : Endpoint<PNMemberArrayResult>
+
+@OptIn(ExperimentalForeignApi::class)
+class GetChannelMembersImpl(
+    private val pubnub: PubNubObjC,
+    private val channel: String,
+    private val limit: Int?,
+    private val page: PNPage?,
+    private val filter: String?,
+    private val sort: Collection<PNSortKey<PNMemberKey>>,
+    private val includeCount: Boolean,
+    private val includeCustom: Boolean,
+    private val includeUUIDDetails: PNUUIDDetailsLevel?
+) : GetChannelMembers {
+    override fun async(callback: Consumer<Result<PNMemberArrayResult>>) {
+        pubnub.getChannelMembersWithChannel(
+            channel = channel,
+            limit = limit?.let { NSNumber(it) },
+            page = createPubNubHashedPage(from = page),
+            filter = filter,
+            sort = sort.map { it.key.fieldName },
+            includeCount = includeCount,
+            includeCustom = includeCustom,
+            includeUUIDFields = includeUUIDDetails == PNUUIDDetailsLevel.UUID,
```

Comment:
shouldn't it be `PNUUIDDetailsLevel.UUID || PNUUIDDetailsLevel.UUID_WITH_CUSTOM` ?